docs/modules/ROOT/pages/reference/extensions/kubernetes.adoc

```diff
@@ -65,6 +65,12 @@ endif::[]
 [id="extensions-kubernetes-additional-camel-quarkus-configuration"]
 == Additional Camel Quarkus configuration
 
+ifeval::[{doc-show-extra-content} == true]
+include::extra-content.adoc[]
+endif::[]
+
+### Automatic registration of a Kubernetes Client instance
```

Comment:
Can we use `=` for the headings because the maven doc tooling will create `id` tags for them. I had not considered the `#` variant.

Reply:
The last commit should fix, I'll rebase before merging if the change is ok.